equinity-historical-data/src/main/java/uk/gov/justice/laa/crime/equinity/historicaldata/model/crm14/Crm14DetailsModel.java

```diff
@@ -842,6 +842,7 @@ public class Crm14DetailsModel extends Crm14AdditionalDetails implements CrmForm
     @JsonProperty("Other_income")
     public float other_income;
 
+    public boolean crm15Section;
```

Comment:
I don't agree with this field. it seems like a redundancy with `hasCrm15`. But without the is/has prefix for booleans.